src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
+
+VERSION_MAJOR = 1
+VERSION_MINOR = 3
+VERSION_REVISION = 0
+VERSION_NUMBER = (VERSION_MAJOR * 1000000 + VERSION_MINOR * 1000 + VERSION_REVISION)
```

Comment:
Those variable are not used by the library itself. Nor they are documented.
If they were used, moving them would be a breaking change, thus maybe an import in client.py could ease the migration.
That being said, I would said that because it's not used nor documented, we should just drop them completely.